server/game/cards/08.3-Km/Jhiqui.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../drawcard.js');
+
+class Jhiqui extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: ({challenge}) =>
+                    challenge.winner === this.controller && this.hasParticipatingLordOrLady()
+            },
+            cost: ability.costs.discardFromHand(),
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card =>
+                    card.location === 'play area' && this.game.currentChallenge.isParticipating(card)
```

Comment:
The target character needs to be a Lord or Lady.

Also, this ability is not a "real" target (doesn't include the word 'choose'), which is taken care of by adding ```type: 'select'``` to target. Lastly, 'Select a character' is the default prompt title so you could get rid of that.